src/engine/list.js

```diff
@@ -8,9 +8,7 @@
   * @param {Array} contents Contents of the list, as an array.
   * @constructor
   */
-function List (name, contents) {
+module.exports = function List (name, contents) {
```

Comment:
The `'func-style': ['error', 'expression']` rule seems to be adding inconsistency to our use of the class pattern here. Before this rule was applied it seemed fairly consistent / clear:
```js
function Foo () {
    // constructor stuff
}

Foo.prototype.bar = function () {
    // other stuff
};

module.exports = Foo;
```